_dashboards/discover/visualizing-ad.md

```diff
@@ -0,0 +1,158 @@
+1. Verify you have selected the appropriate monitor, and then select **Associate monitor**. 
+
+An existing monitor is now associated to the visualization, as shown in the following image:
+
+<insert UI>
+
+## Exploring alerting monitor details
+
+Once you have created or associated alerting monitors, you can verify the monitor is generating the alerts and explore the alert details by following these steps:
+
+1. From the dashboard, view the <**name**> visualization. Alerts are indicated with a red triangle. 
+1. Hover over a triangle to view the alert count details. To investigate the alert details, select a triangle to activate the pop-up window containing the monitor details. Alternatively, select the ellipsis icon in the visualization pane and choose **View events** to view those details.
+1. Select the ellipsis icon, then **Alerting** > **Associated monitors**.
+1. From the Associated monitors window, view the monitor list and then choose an alerting monitor, for example, <**CPU Usage Across World**>. Monitor details such as history, alerts, and associated visualizations, are shown from within the visualization pane.
+1. _Optional_: From the Associated monitors window, unlink a monitor from the visualization by selecting the link icon under **Actions**. This unlinks the monitor from the visualization only; it does not delete the monitor itself.
+1. _Optional_: From the Associated monitors window, edit the monitor's metrics, for example, the threshold for CPU usage. 
+
+Monitor details are viewable from the dashboard and visualization pane, as shown in the following image:
+
+<insert UI>
+
+## Creating anomaly detectors
+
+Anomaly detectors identify unusual patterns or outliers in a dataset. Once  you have defined your problem and prepared your data, you can load it into OpenSearch for indexing. In the following steps, you will continue using the sample dataset that you used in the preceding steps.
+
+To create an anomaly detector using the Discover application and OpenSearch sample data and then visualize the anomalies on a dashboard, follow these steps: 
+
+1. From the OpenSearch Dashboards main menu, select **Discover**.
+1. From the <name> window, select the **<name>** sample dataset. You will see a pre-populated dashboard with line chart visualizations.
+1. Select the ellipsis icon from the **<name>** visualization pane, and then from the **Options** pop-up window, choose **Anomaly Detection** > **Add anomaly detector**.
+1.  Select **Create new detector**.
+1. From the Add anomaly detector window, define the metrics and set thresholds under **Detector details** and **Model features**. To view the visualization within this window, toggle the **Show visualization** button.
```

Comment:
"configure/define the features" may be better wording than "metrics" here. I think thresholds can be removed.

The high-level things the user can set in the creation flow is
- features (which are the AD-term for metric, essentially)
- filters (to filter out unwanted data

Other settings, like timestamp/interval/name/result index, etc. should have reasonable defaults set. And assuming this tutorial is requiring using the sample data, we can confirm that the rest of the detector settings can be pre-populated correctly.